src/components/article-card/article-card.jsx

```diff
@@ -1,32 +1,78 @@
 import "./article-card.css";
 import { useNavigate } from "react-router-dom";
+import * as api from '../../utils/api'
+import axios from "axios";
+import { useState } from "react";
 
 
-const ArticleCard = ({ article }) => {
+const ArticleCard = ({ article, setArticlesList }) => {
+
+  const [selectedArticle, setSelectedArticle] = useState(article)
+  const [disableAdd, setDisableAdd] = useState(false)
+  const [disableRemove, setDisableRemove] = useState(false)
+
+  const addVote = () => {

```

Comment:
I don't understand why this is here when you have an api file with the exact same function, which you have imported.